ci/garbage_collection/cache_cleanups/cache_garbage_collection.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash 
+
+
+    
+. ../garbage_collection_misc.sh
+. ../prune_branches.sh
+
+time_limit=14     ## Days. Set this to give the "deadline". All
+                  ## branches older than this will be removed.
+
+cache_location_dir=/data2/openvdc-ci/branches
```

Comment:
Needs to be changed to `/data` because of the move to dh112. Could be a good idea to read this from the multibox configuration but just hard coding will work for now imo.